snippets/product/type/with-variations/product.liquid

```diff
@@ -25,11 +25,28 @@
             </div>
             <div class="panel-body">
                 <div class="price-block">
-                  {% assign product = product %}
-                  {% include 'product/price' %}
+                    <dl class="dl-horizontal">
+                        <dt class="text-bold">
+                            <span class="pull-left">
+                                    {{ 'products.variation.total_price' | t }}
+                            </span>
+                        </dt>
+                        <dd>
+                            <span class="pull-right text-bold text-green" ng-bind="totalPrice">
+                            </span>
+                        </dd>
+                    </dl>
+                </div>
+
+                <div class="delivery-cart">
+                        <button class="btn btn-lg btn-primary btn-block"
+                        title="{{ 'products.general.add_to_cart' | t }}"
+                        ng-click="addSelectedProductsToCart()">
+                            {{ 'products.general.add_to_cart' | t }}
+                        </button>
                 </div>
 
-                {% include 'product/type/common/product-delivery' %}
```

Comment:
I know that it was shown like this in the design, but i think we should not remove this snippet completely, because it has information about shipping/pickup dates inside. Maybe we should talk with @godothor about if we should leave it or not.

Reply:
It should be so as @godothor said.